WooCommerce/Classes/ViewRelated/Orders/Order Details/Issue Refunds/Cells/RefundShippingDetailsTableViewCell.swift

```diff
@@ -0,0 +1,199 @@
+import UIKit
+
+/// TableViewCell that displays all information regarding shipping refunds
+///
+final class RefundShippingDetailsTableViewCell: UITableViewCell {
+
+    /// Displays a truck image next to the carrier name
+    ///
+    @IBOutlet weak var shippingImageView: UIImageView!
+
+    /// Displays the carrier name and shipping rate
+    ///
+    @IBOutlet weak var carrierLabel: UILabel!
+
+    /// Display the shipping cost
+    ///
+    @IBOutlet weak var shippingPrice: UILabel!
```

Comment:
nit: maybe this can have a name like `shippingPriceLabel` just to be consistent with other `UILabel` properties?